models/docs.md

```diff
@@ -23,7 +23,7 @@ The status of this ad group; see the following [documentation](https://help.ads.
 {% enddocs %}
 
 {% docs ad_name %}
-The name of the corresponding ad; `title_part_1` is used as the ad name as a proxy as one is not provided by the data source.
+The name of the corresponding ad; `title_part_1` is used as the default ad name.
```

Comment:
I think it could be good to note here that `ad_name` can be configured through the new variable you introduced (and then re-gen the docs here and the transform package)

Reply:
good point, updated